### ISSUE TITLE
Decorative ">" symbol in start buttons read out in JAWS

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov_base#427]
In a round of usability testing with screenreader users, we observed that the ">" symbol was read out to someone using JAWS. I've been unable to replicate this in NVDA, Windows Narrator or VoiceOver, but perhaps there is something we can do to avoid this happening in JAWS too?

**Comment by ctorgalson** [comment]
@keelanfh are there specific places this is happening that you can point to?

**Comment by markconroy** [comment]
I _think_ https://github.com/localgovdrupal/localgov_base/pull/594 will fix this.
